direct/data/transforms.py

```diff
@@ -16,6 +16,45 @@
 from direct.utils.asserts import assert_complex, assert_same_shape
 
 
+class ONNX(torch.autograd.Function):
+    """This class is used as a base class for wrapper classes."""
```

Comment:
Is that correct in terms of syntax?

Reply:
I didn't know either, but yes, that's correct